src/components/labware/labware.machine.ts

```diff
@@ -1,34 +1,21 @@
 import { Machine, MachineOptions, send } from 'xstate';
-import {
-  LabwareMachineContext,
-  LabwareMachineEvent,
-  LabwareMachineSchema,
-  Selectable,
-  SelectionMode
-} from './labware.types';
+import { LabwareMachineContext, LabwareMachineEvent, LabwareMachineSchema } from './labware.types';
 import { assign } from '@xstate/immer';
 import { pure } from 'xstate/lib/actions';
-import { SlotFieldsFragment } from '../../types/sdk';
 import { emptySlots, filledSlots, findSlotByAddress, isSlotEmpty, isSlotFilled } from '../../lib/helpers/slotHelper';
 import { sortDownRight } from '../../lib/helpers/labwareHelper';
 
-interface CreateLabwareMachineParams {
```

Comment:
Why has this interface been deleted?

Reply:
This interface was to define the structure of params for createLabwareMachine which we are not using any more